diff --git a/include/fmt/format.h b/include/fmt/format.h
index 287e716..06237be 100644
--- a/include/fmt/format.h
+++ b/include/fmt/format.h
@@ -1268,7 +1268,9 @@ class utf8_to_utf16 {
   }
   inline auto size() const -> size_t { return buffer_.size() - 1; }
   inline auto c_str() const -> const wchar_t* { return &buffer_[0]; }
+#if !defined(__GLIBCXX__) || defined(_GLIBCXX_USE_WCHAR_T)
   inline auto str() const -> std::wstring { return {&buffer_[0], size()}; }
+#endif
 };
 
 enum class to_utf8_error_policy { abort, replace };
